doc/api/http.markdown

```diff
@@ -421,6 +421,10 @@ not be emitted.
 `function (exception, socket) { }`
 
 If a client connection emits an `'error'` event, it will be forwarded here.
+Listener of this event is responsible for destroying socket, and, for example,
+can do it it gracefully by sending 400 HTTP response.
```

Comment:
Maybe re-word this to something like:

```
Listener of this event is responsible for closing/destroying the underlying socket.
For example, one may wish to more gracefully close the socket with an
HTTP 400 Bad Request response instead of abruptly severing the connection.
```